src/app/pages/login/login.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="header-body text-center mb-7">
       <div class="row justify-content-center">
         <div class="col-lg-5 col-md-6">
-          <h1 class="text-white">I Stand With Health Professionals. Are You?</h1>
```

Comment:
Why do you have to take out the "Are You" part?

Reply:
my fault!!!.
I did it to taste my page change and i didn't recognize this change when i push my branch, i will correct it. thanks.